src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/links/LinkInfo.java

```diff
@@ -67,14 +67,14 @@ public abstract class LinkInfo {
     public Content label;
 
     /**
-     * True if we should exclude the type bounds for the type parameter.
+     * True if we should print the type bounds for the type parameter.
      */
-    public boolean excludeTypeBounds = false;
+    public boolean showTypeBounds = true;
```

Comment:
Maybe this and other fields on this class should be encapsulated, maybe with fluent setter methods? Maybe not now, but later?